src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -777,7 +804,6 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			plant.texture = "test";
 			plant.textureBottom = "test2";
 			plant.itemTexture = emptyLists ? "" : "itest";
-			plant.particleTexture = emptyLists ? "" : "test3";

```

Comment:
This should not be removed too. Please check all files, not just test provider for any removed lines so we can rest assured all will work reliably if this gets merged in 2020.6 ;)

Even if you use this later in one of the forks, it is probably good to have the thing working right.